pallets/pallet-did-lookup/src/tests.rs

```diff
@@ -554,3 +557,80 @@ fn test_update_deposit_unauthorized() {
 			);
 		})
 }
+
+// #############################################################################
+// migrate
+
+#[test]
+fn partial_migration() {
+	let deposit_account = || generate_acc32(usize::MAX);
+
+	ExtBuilder::default()
+		.with_balances(vec![(
+			deposit_account(),
+			<Test as crate::Config>::Deposit::get() * 50_000,
+		)])
+		.build()
+		.execute_with(|| {
+			for i in 0..50 {
+				add_legacy_association::<Test>(
+					deposit_account(),
```

Comment:
I did this so that I don't need to care about cloning the address always. It's always annoying that clippy than complains about the last clone not being necessary. Also the clones are a bit verbose in the test cases. `deposit_account()` vs `deposit_account.clone()`

Reply:
Ok, points of view 😊 `AccountId` should implement `Copy` anyway, but I see why you did what you did.